### DISCUSSION TITLE
Help with Sand Simulation Behavior in DualSPHysics

### DISCUSSION THREAD

**Discussion by muratgokoglu35** [DualSPHysics/DualSPHysics#238] [DUALSPHYSICS_V5.2]
Howdy All,

I am working on a simulation involving both water and sand. I've set up my model, but I am having trouble with the sand. When I run the simulation, the sand doesn't behave like sand — it seems to act more like a fluid. Below is an excerpt of the code I am using for defining the sand phase:

![Screenshot 2024-09-26 at 5 38 02 PM](https://github.com/user-attachments/assets/a3d32395-4218-4f31-a042-0f8e2d16969e)
![Screenshot 2024-09-26 at 5 38 33 PM](https://github.com/user-attachments/assets/3c312f32-0c5f-4143-bb7e-9575fe5e27eb)
![Screenshot 2024-09-26 at 5 39 36 PM](https://github.com/user-attachments/assets/3c0526f6-aea5-44e4-aec5-601e3052fc8f)

Any ideas or suggestions on what adjustments I could make to the parameters or simulation settings to improve this? Any help would be greatly appreciated!

Thank you!

**Comment by alexbexe** [comment]
@gfourtakas may help here

**Comment by sandof** [comment]
I'm facing the same problems

**Reply by muratgokoglu3547** [reply]
Hi there, 
I took the DualSPHysics class in November 2024 with Dr. Fourtakas, and he mentioned that sand is not yet available in DualSPHysics. However, we can use Non-Newtonian examples as a substitute( sand-like).
